### PR TITLE
Sle15 sp1 examples fixup

### PR DESCRIPTION
**Pull request by noelpower**
## Various py2/py3 compatibility related problems

* Misc fixes to examples
*  python2 utf8 handling (python2 code no displaying unicode strings)
* incorrect conversion of Py_None to YCP_Null instead of YCP_Void

- [1199243](https://bugzilla.suse.com/show_bug.cgi?id=1199343)
- [1199244](https://bugzilla.suse.com/show_bug.cgi?id=1199344)
- [1199248](https://bugzilla.suse.com/show_bug.cgi?id=1199348)

### CODE REVIEW

File: examples/PatternSelector-wizard-with-packages.py [hunk 1]
```diff
@@ -22,8 +22,7 @@ def main(self):
 
       UI.OpenDialog(
         Opt("defaultsize"),
-        #Wizard("back", "", "cancel", "&Cancel", "ok", "&OK")
-        Wizard(Opt("stepsEnabled"), Id("back"), "", Id("cancel"), "&Cancel", Id("ok"), "&OK")
+        Wizard(Symbol("back"), "", Symbol("cancel"), "&Cancel", Symbol("ok"), "&OK")
```
Comment (dmulder) [inline examples/PatternSelector-wizard-with-packages.py:25]: Why did you remove `Opt("stepsEnabled")`?

File: package/yast2-python-bindings.changes [hunk 2]
```diff
@@ -1,3 +1,13 @@
+-------------------------------------------------------------------
+Thu May  5 11:27:01 UTC 2022 - Noel Power <nopower@suse.com>
+
+- Misc fixes to examples; (bsc#1199343).
+- Fix python2 utf8 handling (python2 code wasn't displaying unicode
+  strings); (bsc#1199348).
+- Change conversion of Py_None to YCP_Null instead of YCP_Void;
```
Comment (dmulder) [inline package/yast2-python-bindings.changes:7]: The change in src/ytypes.i does the opposite of what you describe here in the changes.
Your comment says it changes "Py_None to YCP_Null instead of YCP_Void", but src/ytypes.i changes Py_None from YCPNull to YCPVoid.
Changing it to YCPVoid makes sense (now functions return void instead of NULL), so I think you just need to fix your comment in the changelog.

**Review by dmulder — CHANGES_REQUESTED** [review]

File: examples/PatternSelector-wizard-with-packages.py (hunk 1 above)
Comment (noelpower) [reply examples/PatternSelector-wizard-with-packages.py:25]: because it isn't in the orig Ruby example in ycp-ui-bindings, see the comment I also removed, I think this was an attempt to fix the Symbol/String ambiguity (in the initial work)

File: examples/MultiProgressMeter2.py [hunk 3]
```diff
@@ -110,10 +110,10 @@ def main(self):
 
         if widgetClass == "RadioButton":
           currentUnitID = UI.QueryWidget("unit", "CurrentButton")
-          print "type %s"%type(currentUnitID)
+          print ("type %s"%type(currentUnitID))
           #if Ops.is_term?(currentUnitID):
-          if isinstance (currentUnitID, YCPTerm):
-            unit = currentUnitID.value(0)
+          if isinstance (currentUnitID, tuple):
+            unit = currentUnitID.[0]
```
[the remark this answers is not in the document]
Comment (noelpower) [reply examples/MultiProgressMeter2.py]: fixed (forgot to push this)

**Review by dmulder — APPROVED** [review]
LGTM

**Comment by dmulder** [comment]
@noelpower why are you only submitting to yast:SLE-15-SP1? Shouldn't we also push this to master?

**Comment by noelpower** [comment]
> @noelpower why are you only submitting to yast:SLE-15-SP1? Shouldn't we also push this to master?

in the maint docs it says you should (if you can) fix oldest first and merge forward
